dbtwiz/main.py

```diff
@@ -72,7 +68,7 @@ def build(
     except ValueError:
```

Comment:
Why do we have this date verification in this file rather than in the various command scripts?
If it is to be here, then it should be made into a reusable function rather than duplicating the code for each command.

Reply:
Idea was to do argument validation in the typer command functions, but I'm unsure if this is best practise? Also, we should log errors instead of raising exceptions.